### ISSUE TITLE
Footer GitHub accelerator image pixelates on larger screens 

### ISSUE THREAD

**Issue by ChinmayMhatre** [EddieHubCommunity/LinkFree#6658]
### Description

A higher quality image can be added and a `bg-contain` classname for smaller devices?

![image](https://user-images.githubusercontent.com/51131670/236642709-e4283458-4ddf-4001-87e6-ae9d79e05f2c.png)

![image](https://user-images.githubusercontent.com/51131670/236642719-fcc5a489-5d21-4906-972e-24e064ca072f.png)

**Comment by dhawansolanki** [comment]
Hi, I would like to work on this issue. Thank You.

**Comment by SaraJaoude** [comment]
> Hi, I would like to work on this issue. @SaraJaoude Can you please Assign me this task. Thank You.

@dhawansolanki I will assign this to you. In future please avoid tagging a specific maintainer to request assignment - all maintainers receive notifications and can assign issues.  This makes the project run more smoothly and reduces unnecessary notifications.

**Comment by eddiejaoude** [comment]
No activity for 2 weeks, unblocking by unassigning, so others can move it forwards

**Comment by heyrishabh13** [comment]
Hi, I'd love to contribute in resolving this issue. Can anyone please assign it to me.

**Comment by SaraJaoude** [comment]
> Hi, I'd love to contribute in resolving this issue. Can anyone please assign it to me.

@heyrishabh13 I will assign this to you

**Comment by SaraJaoude** [comment]
@heyrishabh13 how are you getting along with raising a Pull Request for this Issue?

**Comment by SaraJaoude** [comment]
@heyrishabh13 thank you for submitting a Pull Request for this Issue.  As mentioned by @kumarsonsoff3 in the comments for #7537 please remove any updates to your profile (and raise a new Pull Request for these).

**Comment by plbstl** [comment]
i can work on this

**Comment by SaraJaoude** [comment]
> i can work on this

@plbstl I have assigned this to you

**Comment by SaraJaoude** [comment]
As the rebranding has been started this will be fixed as part of that process so I will close this Issue for now.
